Docs/source/building/summit.rst

```diff
@@ -29,6 +29,20 @@ See :doc:`../running_cpp/platforms` for more information on how to run WarpX on
 
 See :doc:`../visualization/yt` for more information on how to visualize the simulation results.
 
+.. note::
+
+   To compile and run WarpX on Summit CPUs, the cuda module is not necessary.
+
+   But to build with the spectral solver, PSATD, you need to use an MPI-enabled version of FFTW like the `fftw` module (because the `cuFFT` tool is not available when cuda is not loaded).
+
+   ::
+
+      module load gcc
+      module load fftw
```

Comment:
With RPaths enabled by default in WarpX, do you need any of those?

Reply:
> With RPaths enabled by default in WarpX

How do you do that? I just followed @LDAmorim's instruction (only `module load fftw`) and it worked for me.

Reply:
I needed, but I didn't test this yet today. @MaxThevenet are you testing it with the current branches versions on Summit?